tutorials/update_brain.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: How to update your robot
```

Comment:
Nit: Should this be "How to update your _brain board_"?

Reply:
That was the original title, but it is too long and wraps wierdly

Reply:
Perhaps "Updating your brain board"? "Brain board updates"?

Reply:
a4aa897